runtime/bindings/python/CMakeLists.txt

```diff
@@ -140,6 +141,8 @@ iree_py_library(
     "iree/_runtime/scripts/iree_benchmark_trace/__main__.py"
     "iree/_runtime/scripts/iree_benchmark_module/__main__.py"
     "iree/_runtime/scripts/iree_cpuinfo/__main__.py"
+    "iree/_runtime/scripts/iree_convert_parameters/__main__.py"
```

Comment:
Note to self: I'm soon going to need to busybox these like on the compiler side. I think we've got like 6 copies of the runtime in our Python distro now (7.9MiB/4.2Mib on Linux/Windows). The only way that has worked this long is that it is pretty binary efficient as it is...

Reply:
Yeah, you could at least combine a few of these (like all the parameter tools) that have the same deps. Things like iree-benchmark-module/iree-run-module/iree-check-module have different primary deps though (gtest/benchmark complicating things).